### ISSUE TITLE
Image format using 256 levels of gray

### ISSUE THREAD

**Issue by hbitteur** [Audiveris/omr-dataset-tools#31]
Could we get gray images (256 gray values on 1 byte) rather than the ARGB color model on 4 bytes.

**Comment by nasehim7** [comment]
Here we are basically talking about saving the png's in Format_GrayScale8 if we have our OMR flag set and not save in ARGB32_Premultiplied. It can be possible with the master using convertToFormat() method of QImage but for MuseScore versions which use Qt version below 5.5, this simple conversion can't be possible. I have done this change for master in my current branch nasehim7/timeSigBranch and we have an active PR: https://github.com/musescore/MuseScore/pull/3669.

Incase, there is anything that I am missing, Kindly let me know :)

**Comment by hbitteur** [comment]
Sorry, I'm not familiar with these C++ names.
The target format (in Java) would be BufferedImage.TYPE_BYTE_GRAY (an unsigned byte grayscale image, non-indexed).
So the word "GrayScale8" sounds (but only sounds) OK to me. That's all I can say.

**Comment by nasehim7** [comment]
Yes Herve I have checked it. Hopefully it will be a fit for tackling our
issue. If not, we will see, what else we can go ahead with. Not an issue. :D
